layouts/index.html

```diff
@@ -0,0 +1,143 @@
+    <meta property='og:description' content='Fast detailed offline maps for travelers, tourists, hikers and cyclists curated with love by MAPS.ME founders' />
+
+    <link rel='canonical'      href='http://organicmaps.app/' />
+    <meta property='og:url' content='http://organicmaps.app/' />
+
+    <meta property='og:site_name' content='Organic Maps' />
+    <meta name='twitter:card' content='summary' />
+    <meta property='twitter:title' content='Organic Maps' />
+    <script type='application/ld+json'>
+    {
+        "@context":"https://schema.org",
+        "@type":"WebSite",
+        "name":"Organic Maps",
+        "headline":"Organic Maps",
+        "description":"Fast detailed offline maps for travelers, tourists, hikers and cyclists curated with love by MAPS.ME founders",
+        "url":"http://organicmaps.app/",
+        "publisher": {
+            "@type":"Organization",
+            "logo": {
+                "@type":"ImageObject",
+                "url":"http://organicmaps.app/android-chrome-512x512.png"
+            }
+        }
+    }
+    </script>
+
+    <link rel='stylesheet' href='main.css'>
+</head>
+
+<body>
+    <h1>Organic Maps</h1>
+    <img src='images/android-chrome-512x512.png' alt='logo'/>
+    <p>
+        <strong>Organic Maps</strong> is an Android &amp; iOS offline maps app for travelers, tourists, hikers, and cyclists based on top of crowd-sourced <strong>OpenStreetMap</strong> data and curated with love by <strong>MAPS.ME</strong> founders.
+    </p>
+    <div id='stores'>
+        <a href='https://apps.apple.com/app/organic-maps/id1567437057'>
+            <img src='images/badges/apple-appstore.png' width='180' />
+        </a>
+        <a href='https://play.google.com/store/apps/details?id=app.organicmaps'>
+            <img src='images/badges/google-play.png' width='180' />
+        </a>
+        <a href='https://appgallery.huawei.com/#/app/C104325611'>
+            <img src='images/badges/huawei-appgallery.png' width='180' />
+        </a>
+        <a href='https://f-droid.org/en/packages/app.organicmaps/'>
+            <img src='images/badges/fdroid.png' width='180' />
+        </a>
+    </div>
+
+    <p float='left'>
+        <img src='images/screenshots/1.jpg' width='400' />
+        <img src='images/screenshots/2.jpg' width='400' />
+        <img src='images/screenshots/3.jpg' width='400' />
+        <img src='images/screenshots/4.jpg' width='400' />
+    </p>
+
+    <h2 id='features'>Features</h2>
+    <p>Organic Maps is the ultimate companion app for travelers, tourists, hikers, and cyclists:</p>
+    <ul>
+        <li>Detailed offline maps with places that don’t exist on other maps</li>
+        <li>Cycling routes, hiking trails, and walking paths</li>
+        <li>Contour lines, elevation profiles, peaks, and slopes</li>
+        <li>Turn-by-turn walking, cycling, and car navigation with voice guidance</li>
```

Comment:
Is it possible to keep original Markdown? Markdowm is easier to edit.

Reply:
ok

Reply:
content/index.md  (or _index.md) should work. The title and content can be easily inserted into the layout template.